e2e/multi-svc-app/front-end/main.go

```diff
@@ -69,13 +71,39 @@ func SetJobCheck(w http.ResponseWriter, req *http.Request, ps httprouter.Params)
 	w.WriteHeader(http.StatusOK)
 }
 
+// PutEFSCheck writes a file to the EFS folder in the container.
+func PutEFSCheck(w http.ResponseWriter, req *http.Request, ps httprouter.Params) {
+	log.Println("Get /efs-putter succeeded")
+	efsVar := os.Getenv("COPILOT_MOUNT_POINTS")
```

Comment:
Is this env var constantized somewhere in the EFS workflow?

Reply:
https://github.com/aws/copilot-cli/blob/9ba39f12ba40837656b196e1f77d445d2f9bb42b/templates/workloads/partials/cf/envvars.yml#L19